src/server/search/search_family.cc

```diff
@@ -230,13 +230,20 @@ optional<SearchParams> ParseSearchParamsOrReply(CmdArgParser parser, ConnectionC
   return params;
 }
 
-std::optional<std::string_view> ParseField(CmdArgParser* parser, bool expect_at_sign = false) {
+std::optional<std::string_view> ParseField(CmdArgParser* parser) {
```

Comment:
I did not say - to separate into two functions but if you did that 
then no need for optional here - you always return string_view